pkg/operator/mon/health.go

```diff
@@ -143,9 +143,43 @@ func (c *Cluster) checkHealth() error {
 		return nil
 	}
 
+	// check if there are more than two mons running on the same node, failover one mon in that case
+	ret, err := c.checkMonsUsed()
```

Comment:
a couple naming changes you might consider:
- `ret`: might read better as `done`
- `checkMonsUsed`: perhaps `checkMonsOnSameNode`